sway/commands/resize.c

```diff
@@ -173,8 +173,10 @@ void container_resize_tiled(struct sway_container *con,
 			return;
 		}
 
+		// Use an extra 0.1 pixels for the calculation to avoid rounding down
+		// by the layout code eating a pixel
 		double amount_fraction =
-			((double)amount / con->width) * con->width_fraction;
+			(((double)amount + 0.1) / con->width) * con->width_fraction;
```

Comment:
Would be nice to find a better way to fix this…

Reply:
Using `round()` instead of `floor()` in the layout code would eliminate the need for that but would make it possible in extreme cases for the children to end up larger than the parent.

Depending on how correct the layout code should be things could get even more complex. For example right now the last container always gets whatever leftover space there is from doing `floor()` to all the other widths. That means that if you resize that last container to a fixed px size it won't be correctly sized.

If that kind of pixel correctness is not wanted then maybe PR isn't needed at all.